src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -347,3 +347,59 @@ def result(data: Union[Node, Output, NumericData], name: Optional[str] = None) -
     if isinstance(data, Node):
         return Result(data.output(0))
     return Result(data)
+
+
+@nameable_op
+@apply_affix_on("data", "input_low", "input_high", "output_low", "output_high")
+def fake_quantize(
+    data: NodeInput,
+    input_low: NodeInput,
+    input_high: NodeInput,
+    output_low: NodeInput,
+    output_high: NodeInput,
+    levels: int,
+    auto_broadcast: str = "NUMPY",
+    name: Optional[str] = None,
```

Comment:
why we decided not to use "name" as prefix? 
is "fake_quantize" the only operation where we create internal auxiliary operations? do we plan to update them?

Reply:
This should partially answer the question: https://github.com/openvinotoolkit/openvino/pull/22327#discussion_r1466305358

I decided kept it as robust as possible so all names can be accepted depending on needs of developers. It's public API of operators overall so forcing a concept of names is not up to me to decide. It can be easily changed in future with small changes to current logic.

> is "fake_quantize" the only operation where we create internal auxiliary operations? do we plan to update other ops?

If any other operator is deemed to be extended, feel free to create PR by yourself (especially to be reviewed by opset team , cc @mitruska ) or create a list of such operators and pass it down to me. With current decorator approach it's really easy to apply, just copy-paste argument names and add info to docstring (can be copied from `fake_quantize`).

Reply:
Ok but in case when we use `name` as a prefix, it can be  automatically  applied to  all existing ops, without updating them and adding to a new opset. The current solution won't solve the issue for the previous opsets.

Maybe we can use 'name' as prefix for all internally created ops at least?
 I mean

opset6.fake_quantize(data, [1,2,3], [1,2,3], [1, 2,3],)  // just for example

these [1,2,3] tensors will be internally (inside python function) converted to ov::Constant operation, and will have automatically generated names, e.g. Constant_N. So we can fix these names without breaking backward compatibility.  

So for user it will look like

opset6.fake_quantize(data, [1,2,3], [1,2,3], [1, 2,3], "fq_name")  // just for example
all Constants will have "fq_name" + Constant_N and most probably the names will be unique.